go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/minio/mc
 
-go 1.19
+go 1.21
+
+toolchain go1.21.5
```

Comment:
Revert.

This is not a "desired version", this is the minimum version, and it should not be 1.21

Reply:
thanks, removed.